### PR TITLE
Rbac

### PR DESCRIPTION
**Pull request by AVaitkunas**
Implemented role based auth system. Currently two roles user and admin are supported.

### CODE REVIEW

File: kw_manager.go [hunk 1]
```diff
@@ -43,16 +43,16 @@ func main() {
 	app.Get(routes.TechnicalKwdsUrl, routes.Handler(router.HandleTechnicalKeywords))
 	app.Get(routes.AllKwdsUrl, routes.Handler(router.HandleAllKeywords))
 
-	app.Get(routes.CreateKwdUrlFull, routes.Handler(router.HandleCreateKeywordGet))
-	app.Post(routes.CreateKwdUrlFull, routes.Handler(router.HandleCreateKeywordPost))
+	app.Get(routes.CreateKwdUrlFull, auth.RolesRequires(database.RoleAdmin), routes.Handler(router.HandleCreateKeywordGet))
```
Comment (AVaitkunas) [inline kw_manager.go]: should we start splitting routes? because lines length is getting too big?

Comment (AngelVI13) [reply kw_manager.go]: yeah put line breaks

File: pkg/database/models.go [hunk 2]
```diff
@@ -13,8 +13,9 @@ import (
 type RoleType string
 
 const (
-	RoleUser  RoleType = "user"
-	RoleAdmin          = "admin"
+	RoleUser      RoleType  = "user"
+	RoleAdmin               = "admin"
+	RoleAnonimous           = "anonimous"
```
Comment (AngelVI13) [inline pkg/database/models.go]:
```suggestion
	RoleAnonymous           = "anonymous"
```

File: pkg/routes/index.go [hunk 3]
```diff
@@ -1,8 +1,30 @@
...
+
 func (r *Router) HandleIndex(c *Ctx) error {
 	data := c.FlashData()
 	data["Title"] = "Keyword storage"
 
+	businesskwds, err := database.BusinessKeywords(r.db)
+	if err != nil {
+		return c.WithError(fmt.Sprintf(
+			"error while fetching business keywords: %v", err),
+		).Render(KeywordsView, data)
```
Comment (AngelVI13) [inline pkg/routes/index.go]:
```suggestion
		).Render(IndexView, data)
```

same for other error

Comment (AVaitkunas) [reply pkg/routes/index.go]: fixed

File: pkg/database/models.go (hunk 2 above)
Comment (AVaitkunas) [reply pkg/database/models.go]: fixed

File: kw_manager.go (hunk 1 above)
Comment (AVaitkunas) [reply kw_manager.go]: fixed

**Review by AngelVI13 — APPROVED** [review]

**Pull request merged**